src/resources/views/trongate.blade.php

```diff
@@ -9,6 +9,12 @@
     <h1>You've made a <u>huge</u> mistake for using Laravel.</h1>
     <h2>Use Trongate!</h2>
 
-    <p>You will be redirected in <span _="on load set $timer to 5 repeat 5 times decrement $timer then put it into my innerHTML wait 1s end then go to url 'https://trongate.io'">5</span> seconds.</p>
+    <p>
+        You will be redirected in
+        <span _="on load set $timer to {{ config('trongate.countdown') }} repeat {{ config('trongate.countdown') }} times decrement $timer then put it into my innerHTML wait {{ config('trongate.redirect_delay') }}s end then go to url 'https://trongate.io'">
```

Comment:
Please revert the `wait 1s` since this is the interval for each iteration. This should not be configurable since it does not make much sense.